controller/api/public/stat_summary.go

```diff
@@ -329,6 +459,37 @@ func buildRequestLabels(req *pb.StatSummaryRequest) (labels model.LabelSet, labe
 	return
 }
 
+func buildTrafficSplitRequestLabels(req *pb.StatSummaryRequest) (labels model.LabelSet, labelNames model.LabelNames) {
+	// labelNames: the group by in the prometheus query
+	// labels: the labels for the resource we want to query for
+
+	switch out := req.Outbound.(type) {
+	case *pb.StatSummaryRequest_ToResource:
+		labelNames = promGroupByLabelNames(req.Selector.Resource)
```

Comment:
The selector resource is the traffic split object itself, right?  so I don't think we want to construct any labels based on that object.